packages/app-bridge/src/AppBridgePlatformApp.ts

```diff
@@ -0,0 +1,43 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { ErrorMessageBus, IMessageBus, MessageBus } from './utilities/MessageBus';
+import { generateRandomString, notify, subscribe } from './utilities';
+import { MethodsListKeys, PlatformAppMethods, Topic } from './types';
+import { InitializationError } from './errors/InitializationError';
+import { getQueryParameters } from './utilities/queryParams';
+
+const PUBSUB_CHECKSUM = generateRandomString();
+
+type InitializeEvent = {
+    port: MessagePort;
+};
+
+export class AppBridgePlatformApp {
+    private messageBus: IMessageBus = new ErrorMessageBus();
```

Comment:
```suggestion
    private errorMessageBus: IMessageBus = new ErrorMessageBus();
```

Reply:
I think this pattern is intentional. When any of the methods (`.api()`, `.context()`) are called without prior initialization, we fall back to the default `ErrorMessageBus`, otherwise we use `MessageBus`.